packages/pod-install/src/index.ts

```diff
@@ -41,6 +42,26 @@ async function runAsync(): Promise<void> {
 
   const possibleProjectRoot = CocoaPodsPackageManager.getPodProjectRoot(projectRoot);
   if (!possibleProjectRoot) {
+    try {
+      const jsonData = JSON.parse(readFileSync(join(projectRoot, 'package.json')).toString());
+      const hasExpo = jsonData.dependencies?.hasOwnProperty('expo');
+      const hasReactNativeUnimodules = jsonData.dependencies?.hasOwnProperty(
+        'react-native-unimodules'
+      );
+      const isManaged = hasExpo && !hasReactNativeUnimodules;
+
+      if (isManaged) {
+        info(
+          chalk.yellow(
+            'In a managed project, `pod-install` will run automatically as part of the EAS Build process. Learn more: https://docs.expo.dev/build-reference/ios-builds/'
```

Comment:
```suggestion
            `Skipping ${chalk.bold('pod install')} because no CocoaPods project found. In Expo projects, the ${chalk.bold('ios')} directory can be generated with ${chalk.bold('npx expo prebuild')} when needed. Learn more: https://docs.expo.dev/workflow/customizing/'
```

we should copy over `learnMore` from https://github.com/expo/expo-cli/blob/97e299f216a71bd694a47c29f33d15da7c7185e6/packages/expo-cli/src/commands/utils/TerminalLink.ts#L33-L44 and use that here too